src/Umbraco.Web.UI.Client/src/packages/media/media/components/input-image-cropper/input-image-cropper.element.ts

```diff
@@ -67,6 +72,18 @@ export class UmbInputImageCropperElement extends UmbFormControlMixin<
 
 	protected override firstUpdated(): void {
 		this.#mergeCrops();
+		this.#observeAcceptedFileTypes();
+	}
+
+	async #observeAcceptedFileTypes() {
+		await this.#temporaryFileConfig.initialized;
```

Comment:
@iOvergaard Sorry, I commented on the wrong line, it's this one that doesn't resolve.

The strange thing is that it works in other places, e.g. the `umb-property-editor-ui-accepted-upload-types` component. I can't understand why it doesn't work in the image cropper element. 🤔